### DISCUSSION TITLE
Unable to Prove Student Status for Developer Pack

### DISCUSSION THREAD

**Discussion by tsouchlarakis** [community/community#110841] [GitHub Education]
### Select Topic Area

General

### Body

I am trying to upload proof that I am enrolled in an academic institution at https://education.github.com/pack/join

I've tried:
- taking a webcam picture of the front of my student ID → problem: no date present
- taking a webcam picture of the back of my student ID → problem: no name present

Since the name is on the front and the date is on the back, and the picture tool doesn't allow multiple photos, I'm not able to get past this step.

The page also mentions that it should be possible to upload proof, but there is no place to upload anything. There is no file upload area or button to accomplish this.

Because of this, I'm unable to prove my status as a student.

**Comment by SebastienXia** [comment]
Hi, @tsouchlarakis 

Normally, there is a button to download a file in PNG or JPG, to be able to approve your membership to a school.

"below the take to picture area"

**Comment by tsouchlarakis** [comment]
Am I missing something?
<img width="1013" alt="Screenshot 2024-03-04 at 9 43 50 AM" src="https://github.com/community/community/assets/26070649/5d27e5c2-4b10-4313-829d-3b2656080ab1">

**Reply by sadekur** [reply]
respons #110841

**Comment by SebastienXia** [comment]
look at this video : https://youtu.be/iyThW3RvZpk?si=BHpX_hKQkHxsxrUA
The person has the button to download a file. try to refresh idk

**Comment by tsouchlarakis** [comment]
Maybe they updated that page since the video was made and the video is out of date. From that video:

<img width="547" alt="Screenshot 2024-03-04 at 11 15 57 PM" src="https://github.com/community/community/assets/26070649/8ec9991f-55a9-4334-902b-d120a24daaff">

And from the actual page:

<img width="761" alt="Screenshot 2024-03-04 at 11 16 01 PM" src="https://github.com/community/community/assets/26070649/83e8c269-9720-447d-b655-28471d265a1a">

Presently, there is no "Upload an image" button like it shows in the video. I've tried both Safari and Chrome.

**Comment by DontBlameMe99** [comment]
This is exactly my problem as well. I there any fix?

**Comment by davevad93** [comment]
Hi, I had the same problem to get my student benefits. They rejected my request 2 times but I solved it by uploading an image of my exam notes which had a date showing that I was studying. I was able to do this because my school has an online platform where they upload all the student records. As an alternative, you could ask your high school principal to write a letter certifying that you are currently studying, take a picture of it and then upload it to GitHub.
